File: Day 98/homework/hw5.py
# 5) შექმენით Tuple სახელწოდებით info, სადაც შეინახავთ თქვენს მონაცემებს (სახელი, ასაკი, მისამართი ა.შ). მასში შეინახეთ 5 მონაცემი და მოახდინეთ თაფლის unpacking: თითოეულ მონაცემს შესაბამისი ცვლადის სახელი შეისაბამეთ (name, age, etc..)


info = ("gvanca", 'mazanishvili', '15 years old', '10 grader', 'leader')
name, lastname, age, grade, role = info

print(name)
print(lastname)
print(age)
print(grade)
print(role)

File: Day 150/homework/hw5.js
// 5) შექმენით ობიექტი updatedStudent, რომელიც იქნება student ობიექტის ასლი, მაგრამ მასში grade იქნება შეცვლილი (მაგალითად: "A") და დამატებული ექნება ახალი თვისება isGraduated: true.

const student = {
    name: "Gvanca",
    surname: "Mazanishvili",
    grade: "C"
}



const updatedStudent = {
    ...student, 
    grade: "A",
    isGraduated: true
}

console.log(updatedStudent);